pkg/server/registry/apigroups/acorn/projects/storage.go

```diff
@@ -1,37 +1,76 @@
 package projects
 
 import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/acorn-io/baaah/pkg/router"
 	"github.com/acorn-io/mink/pkg/stores"
+	"github.com/acorn-io/mink/pkg/strategy"
 	"github.com/acorn-io/mink/pkg/strategy/remote"
 	"github.com/acorn-io/mink/pkg/strategy/translation"
+	"github.com/acorn-io/mink/pkg/types"
 	apiv1 "github.com/acorn-io/runtime/pkg/apis/api.acorn.io/v1"
+	v1 "github.com/acorn-io/runtime/pkg/apis/internal.acorn.io/v1"
+	"github.com/acorn-io/runtime/pkg/labels"
 	"github.com/acorn-io/runtime/pkg/tables"
 	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apiserver/pkg/registry/rest"
 	kclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func NewStorage(c kclient.WithWatch) rest.Storage {
-	remoteResource := translation.NewTranslationStrategy(&Translator{apiv1.LocalRegion},
-		remote.NewRemote(&corev1.Namespace{}, c))
-	strategy := &Strategy{
-		c:       c,
-		lister:  remoteResource,
-		creater: remoteResource,
-		updater: remoteResource,
-		deleter: remoteResource,
-	}
-
-	validator := &Validator{DefaultRegion: apiv1.LocalRegion, Client: c}
+	remoteResource := translation.NewSimpleTranslationStrategy(&translator{}, remote.NewRemote(&v1.ProjectInstance{}, c))
+	validator := &Validator{Client: c}
 	return stores.NewBuilder(c.Scheme(), &apiv1.Project{}).
-		WithCreate(strategy).
-		WithUpdate(strategy).
-		WithDelete(strategy).
-		WithGet(strategy).
-		// Watch is not enabled because the dynamic privileges can do weird things...
-		WithList(strategy).
+		WithCompleteCRUD(remoteResource).
+		WithCreate(&projectCreater{creater: remoteResource, client: c}).
 		WithValidateCreate(validator).
 		WithValidateUpdate(validator).
 		WithTableConverter(tables.ProjectConverter).
 		Build()
 }
+
+type projectCreater struct {
+	creater strategy.Creater
+	client  kclient.Client
+}
```

Comment:
nit: I know the underlying typo/misspelling comes from mink, but I think this is worth changing anyway

```suggestion
type projectCreator struct {
	creator strategy.Creater
	client  kclient.Client
}
```

(Would need to modify other parts of this file too.)